src/snowflake/cli/plugins/nativeapp/run_processor.py

```diff
@@ -145,8 +145,8 @@ def _execute_sql_script(self, sql_script_path):
             sql_script = f.read()
 
         try:
-            if self._conn.database:
-                self._execute_query(f"use database {self._conn.database}")
+            if self.app_name:
+                self._execute_query(f"use database {self.app_name}")
```

Comment:
Update the doc comment to reflect this change too

Reply:
the doc has been updated in a previous PR.